src/workloads/execution/ShardedLookup.yml

```diff
@@ -111,6 +111,62 @@ Actors:
         # To get meaningful results, the entire result set should fit in a single batch. This should
         # be possible since both collections are small.
         cursor: {batchSize: *NumDocs}
+    - OperationMetricsName: TargetedLookupShardedIntoSharded # PERF-2359
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: Collection0
+        pipeline:
+          [
+            {$lookup: {
+              from: "Collection1",
+              localField: "shardKey",
+              foreignField: "shardKey",
+              as: "joined"
+            }}
+          ]
+        cursor: {batchSize: *NumDocs}
+    - OperationMetricsName: UntargetedLookupShardedIntoSharded # PERF-2359
```

Comment:
Copy-paste error? The untargeted sharded to sharded workload already exists (line 89), PERF-2359 refers to the targeted version you have above on line 114.

Reply:
No this was purposeful. I wanted to get an apples to apples comparison to the targeted sharded to sharded workload, without modifying Hana's existing workload. But you're right, technically this is duplicated work.